docs/Project-Configuration/Project-Commands.md

```diff
@@ -905,6 +905,35 @@ Parameters:
     internally.
 -   `destination`: expansion name to save the value to.
 
+## papertrail.trace
+
+This command traces artifact releases with the Papertrail service.
```

Comment:
Can you add a mention here that this is maintained by your team, and what channel to ask questions to? (This is what we did for mac.sign before that command was removed)

Reply:
Done!